ContentReferenceExtensions.cs

```diff
@@ -157,9 +156,9 @@ public static string GetFriendlyUrl(string url, GetFriendlyUrlOption options = G
             return resolved.Href;
         }
 
-        public static string? ResolveUrl(this IContent content, LinkOptions options = LinkOptions.None)
+        public static string? ResolveUrl(this IContent content, string languageBranchId = "", LinkOptions options = LinkOptions.None)
```

Comment:
Would it be worthwhile to also add the language branch ID to the `public static string? ResolveUrl(this ContentReference content, LinkOptions options = LinkOptions.None)` extension method below this? It would at least be a bit more consistent.

Reply:
I agree with this change for sure.